scripts/src/main/resources/scripts/command/intellij

```diff
@@ -14,49 +14,32 @@ then
   exit
 fi
 
-if [ -n "${DEVON_IDE_TRACE}" ]; then set -vx; fi
+# shellcheck source=scripts/functions
+source "$(dirname "${0}")"/../functions
 IDEA_HOME="${DEVON_IDE_HOME}/software/intellij"
-if [ "${OSTYPE:0:6}" = "darwin" ]
+if doIsMacOs
 then
-  TOOL_VERSION_COMMAND="cat '${IDEA_HOME}/IntelliJ.app/Contents/Resources/product-info.json'"
+  # TODO path needs to be fixed...
+  TOOL_VERSION_COMMAND="cd '${DEVON_IDE_HOME}/software/macos/intellij'/*.app && head -n 6 'Contents/Resources/product-info.json'"
 else
-  TOOL_VERSION_COMMAND="cat '${IDEA_HOME}/product-info.json'"
+  TOOL_VERSION_COMMAND="head -n 6 '${IDEA_HOME}/product-info.json'"
 fi
-# shellcheck source=scripts/functions
-source "$(dirname "${0}")"/../functions
+# shellcheck source=scripts/commandlet-cli
+source "$(dirname "${0}")"/../commandlet-cli
 cd "${DEVON_IDE_HOME}" || exit 255
 
 function doSetup() {
   doDevonCommand java setup silent
-  if [ "${1}" != "silent" ] || [ ! -d "${IDEA_HOME}" ]
+  doInstall "intellij" "${INTELLIJ_VERSION}" "${1}" "${INTELLIJ_EDITION_TYPE:-C}"
+  if [ "${?}" = 0 ]
```

Comment:
The download mirrors

- https://ftp.osuosl.org/idea/ideaIC-2022.2.3.tar.gz
- https://mirror.math.princeton.edu/idea/ideaIC-2022.2.3.tar.gz

don't work anymore.

Reply:
See here:
https://github.com/devonfw/ide-mirrors/blob/master/intellij/urls

It seems that these two mirrors dropped idea so we need to remove them.